thoth/storages/graph/dgraph.py

```diff
@@ -2029,6 +2051,47 @@ def sync_analysis_result(self, document: dict) -> None:
         self._python_sync_analysis_result(package_extract_run, document, environment)
         self._python_file_digests_sync_analysis_result(package_extract_run, document)
 
+    def _python_sync_package_analysis_result(self, package_analyzer_run: PackageAnalyzerRun, document: dict) -> None:
+        """Sync results of Python packages found in the given artifact."""
+        python_package_version = PythonPackageVersion.from_properties(
+            ecosystem="python",
+            package_name=document["metadata"]["arguments"]["python"]["package_name"],
+            package_version=document["metadata"]["arguments"]["python"]["package_version"],
+            index_url=document["metadata"]["arguments"]["python"]["index_url"],
+            os_name=document["metadata"]["distribution"]["id"],
+            os_version=document["metadata"]["distribution"]["version"],
+            python_version=document["metadata"]["python"]["api_version"],
```

Comment:
@fridex I just put it. However what I could do here is concatenate major and minor from this json```"python": {
      "api_version": 1013,
      "implementation_name": "cpython",
      "major": 3,
      "micro": 3,
      "minor": 6,
      "releaselevel": "final",
      "serial": 0
    }```